core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/error/CustomGlobalExceptionHandler.java

```diff
@@ -31,7 +32,17 @@ public void handleMetadataAlreadyExistsException(HttpServletResponse response) {
 	@ExceptionHandler(DataBadRequestException.class)
 	public void HandleBadRequest(HttpServletResponse response) throws IOException {
 		response.sendError(HttpStatus.BAD_REQUEST.value());
+	}
 
+	@ResponseStatus(HttpStatus.BAD_REQUEST)
+	@ExceptionHandler(SwaggerParserException.class)
+	@ResponseBody
+	public Map<String, String> handleSwaggerParserException(SwaggerParserException e) {
+		Map<String, String> errMessages = new HashMap<>();
+		errMessages.put("error", "Something is wrong in your documentation");
+		errMessages.put("errorCode", "400");
+		errMessages.put("message", e.getMessage());
```

Comment:
concatenate the list of error messages like: "we found the following errors: ..."